region/brinstar/red/Caterpillar Room.json

```diff
@@ -960,7 +977,15 @@
         "canCrouchJump",
         {"enemyDamage": {"enemy": "Zero", "type": "contact", "hits": 1}}
       ],
-      "note": "Let the Caterpillar climb the right wall and jump on top of it in Morph Ball form to bounce up to the Cacatac platform."
+      "wallJumpAvoid": true,
+      "flashSuitChecked": true,
+      "note": [
+        "Let the Zero (Caterpillar) climb the right wall and jump on top of it in Morph Ball form to bounce up to the Cacatac platform."
+      ],
+      "detailNote": [
+        "One setup is to do a regular jump while aiming down to normalize the camera and then let the Zero move off camera.",
+        "And then crouch jump directly into it."
```

Comment:
```suggestion
        "One setup is to do regular jumps while aiming down until the camera stops lowering, and then let the Zero move off camera.",
        "And then crouch jump and morph directly into it, while holding right."
```

Reply:
It usually took me 3 jumps to get the camera low enough. 
This detail could also be added to all of the g-mode setups that use it as well.